libs/api/domains/cms/src/lib/models/news.model.ts

```diff
@@ -32,12 +35,13 @@ export class News {
 }
 
 export const mapNews = ({ fields, sys }: INews): News => ({
+  typename: 'News',
   id: sys.id,
   slug: fields?.slug ?? '',
   title: fields?.title ?? '',
   subtitle: fields?.subtitle ?? '',
   intro: fields?.intro ?? '',
   image: mapImage(fields.image),
```

Comment:
```suggestion
  image: fields.image?.fields?.file ? mapImage(fields.image) : null,
```

Reply:
```mapImage(fields.image ?? '')``` 
works fine as well

Reply:
`mapImage` is quite bulletproof atm.
We can assume fields is allways set so, worst case is fields.image will pass undefined which is handled inside mapImage.
We are providing empty image object instead of null atm since null is probably not handled in frontend.
It makes sense to pass null once that has been handled, but we will tackle that as a separate issue